.github/workflows/blank.yml

```diff
@@ -0,0 +1,14 @@
+name: Labeler
+on: [issues]
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - uses: damccorm/tag-ur-it@master
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        configuration-path: "./issue-rules.yml"
```

Comment:
any way this could read the code owners file?

Reply:
I think they're actually 2 seperate concepts. Codeowners assigns people based on area paths touched, this assigns people based on issue content. So while I think _maybe_ a mapping could make sense, its not immediately clear to me what that would look like or that it makes sense in the general case (I'd like to keep the action extensible if possible)

I think I'd probably prefer to take that as a potential future enhancement if that's ok.

Reply:
gotcha, sounds good